src/pages/HomePage.vue

```diff
@@ -1,42 +1,39 @@
 <template>
-  <q-page class="flex column justify-between home-page">
-    <div class="fit flex justify-center items-center home-banner">
-      <q-img
-        src="icons/logo_modelizer.svg"
-        class="home-logo"
-        :ratio="1"
-      />
-      <h2 class="text-white">
-        {{ $t('application.name') }}
-      </h2>
-    </div>
-    <div class="column items-center home-content">
-      <div class="fit row justify-center">
-        <project-grid
-          class="col-md-8"
-          :projects="projects"
-        />
-      </div>
-      <div class="fit row justify-center q-mt-lg">
-        <template-grid
-          class="col-md-8"
-          :templates="templates"
-          @add:template="openCreateProjectTemplateDialog"
-        >
-          <template #header>
-            <h4>
-              {{ $t('page.home.template.createProject') }}
-            </h4>
-          </template>
-        </template-grid>
-      </div>
-    </div>
-    <import-project-dialog />
-    <create-project-template-dialog />
-    <create-project-dialog />
-    <delete-project-dialog />
-    <rename-project-dialog />
-  </q-page>
+  <q-layout
+    class="home-page"
+  >
+    <q-page-container>
+      <q-page class="home-page">
+        <default-navigation-bar />
```

Comment:
according to [quasar layout](https://quasar.dev/layout/header-and-footer) q-header are outside `q-page-container` and have to be put inside `q-layout` as first element.

So like that you can remove the `padding-top: 0 !important;`

Reply:
done